src/kirby/components/list/list-cell/list-cell.component.scss

```diff
@@ -0,0 +1,15 @@
+.cell {
+    height: 100%;
```

Comment:
Looks like your editor is set to have tabs 4 spaces long and not 2 like the rest of the solution?